### PR TITLE
Cluedo - Nathalie 

### PR DESCRIPTION
**Pull request by nathalienordstrom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+
+I have a hard time to write the javascript from scratch by myself at this point. Lucky for me you guys had started the Cluedo game for me and I started to put in all the info in the arrays and objects. Then I search and found some code that fits the project and I tried to turn the code to ”my own”. 
+I put in picture for the weapons and styled the page with a blood button and sm frames for all the diffrent images. If i would have more time i would try to manage the red and black level.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://app.netlify.com/sites/cluedo-nathalie/settings/general.
```
Comment (nathalienordstrom) [inline README.md]: https://cluedo-nathalie.netlify.app

File: code/clue.js [hunk 2]
```diff
@@ -40,20 +169,72 @@ const randomSelector = array => {
...
+const theKillerImage = document.getElementById("killerImage")
+const theKillerDescription = document.getElementById("killerDescription")
+
+theKiller.style.background = mystery.killer.color
+theKillerName.innerHTML = mystery.killer.firstName + " " + mystery.killer.lastName
+theKillerAge.innerHTML = mystery.killer.age
+theKillerOccupation.innerHTML = mystery.killer.occupation
+theKillerImage.setAttribute("src", mystery.killer.image)
+theKillerDescription.innerHTML = mystery.killer.description
 }
```
Comment (ckjerrgren) [inline code/clue.js:191]: I think that if you had used the function that was suggested by the teachers 
```
  document.getElementById( 'killerName').innerHTML =`${mystery.killer.firstName}${mystery.killer.lastName}`
```
you could have gotten away with writing less code. Don't know if it was necessary to set the variables(objects?) as const first.

Comment (ckjerrgren) [reply code/clue.js:191]: Same with weapon and room :)

File: code/clue.js [hunk 3]
```diff
@@ -40,20 +169,72 @@ const randomSelector = array => {
...
+  const theRoom = document.getElementById("room")
+  const theRoomName = document.getElementById("roomName")
+
+  theRoomName.innerHTML = mystery.room
+
+}
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
 // 'The murder was committed by Jacob Green, in the living room with a rope.'
+
+const revealMystery = () => {
```
Comment (ckjerrgren) [inline code/clue.js:226]: Nice use of if and else from last week

File: code/index.html [hunk 4]
```diff
@@ -1,37 +1,59 @@
...
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <link rel="shortcut icon" type="image/png" href="favi.png">
```
Comment (ckjerrgren) [inline code/index.html:9]: I like the favicon

File: code/style.css [hunk 5]
```diff
@@ -1,58 +1,214 @@
...
+  outline: none;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: 16px auto;
+  cursor: pointer;
+  box-shadow: 10px 10px 10px black;
+}
+
+button:hover {
```
Comment (ckjerrgren) [inline code/style.css:181]: Wow!

File: code/style.css [hunk 6]
```diff
@@ -1,58 +1,214 @@
...
+  background: url("frame-2.png");
+  background-size: 100% 100%;
+}
+
+.wdeck p{
+  font-size: 13px;
+  margin-bottom:0px;
+}
+
+.weapon-deck img {
```
Comment (ckjerrgren) [inline code/style.css:112]: Some of the weapon pictures takes a bit longer to load than the rest.
Are some of them larger?

**Review by ckjerrgren — COMMENTED** [review]
Very nice website! 

I like all the little extra effects you've added and that it is responsive. 
Very readable code but you could have added your own comments to make it even easier for others to read and understand.

File: code/clue.js [hunk 7]
```diff
@@ -5,16 +5,114 @@ const mrGreen = {
...
+  weight: 80,
+  feature: 'shiny',
+  image: 'assets/trophy.png',
+}
+const pistol = {
+  name: 'pistol',
+  weight: 60,
+  feature: 'leathal',
+  image: 'assets/pistol-.png',
+
```
Comment (EmOlsz) [inline code/clue.js]: Unnecessary blank line

File: code/clue.js [hunk 8]
```diff
@@ -40,20 +169,72 @@ const randomSelector = array => {
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+
```
Comment (EmOlsz) [inline code/clue.js]: It's far more clean to initialise our `mystery` object already with properties that are going to be there at the end, but with initial value of `null`, like that :
```
const mystery = {
  killer: null,
  weapon: null,
  room: null,
}
```

File: code/clue.js [hunk 9]
```diff
@@ -40,20 +169,72 @@ const randomSelector = array => {
...
   mystery.killer = randomSelector(suspects)
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+const theKiller = document.getElementById("killer")
+const theKillerName = document.getElementById("killerName")
+const theKillerAge = document.getElementById("killerAge")
```
Comment (EmOlsz) [inline code/clue.js]: Wrong indentation everywhere in this function

File: code/clue.js [hunk 10]
```diff
@@ -40,20 +169,72 @@ const randomSelector = array => {
...
 }
 
+
 // CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+const pickWeapon = () => {
+  // This will randomly select a weapon. And add that to the mystery object.
+  mystery.weapon = randomSelector(weapons)
+
+  
+const theWeapon = document.getElementById("weapon")
```
Comment (EmOlsz) [inline code/clue.js]: We don't need prefix `the` in every variable - let's delete that

File: code/style.css [hunk 11]
```diff
@@ -1,58 +1,214 @@
...
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  height: auto; 
+  background: transparent;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  flex-wrap: wrap;
+  background-image: url(assets/cluedo_cover.jpeg);
```
Comment (EmOlsz) [inline code/style.css]: Let's wrap it up with quote marks, like that :
```
background-image: url("assets/cluedo_cover.jpeg");
```
Some browsers might not approve this value without quote marks

File: code/style.css [hunk 12]
```diff
@@ -1,58 +1,214 @@
...
+.kdeck {
+  width: 280px;
+  height: 440px;
+  text-align: center;
+  padding-top: 40px;
+  background: url("frame.png");
+  background-size: 100% 100%;
+}
+
+#killer.killer-deck {
```
Comment (EmOlsz) [inline code/style.css]: We should avoid styling by ids, let's change that to class reference

File: code/style.css [hunk 13]
```diff
@@ -1,58 +1,214 @@
...
+  margin-top: 20px;
 }
 
-.weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+.kdeck p{
+  font-size: 16px;
+  margin-bottom:0px;
```
Comment (EmOlsz) [inline code/style.css]: No need to add `px` when value is equal `0`. `margin-bottom: 0;` is completely ok

File: code/style.css [hunk 14]
```diff
@@ -1,58 +1,214 @@
...
+  opacity: 50%;
+  color: black;
+  border-radius: 16px;
+  box-shadow: 10px 10px 10px darkred;
+}
+
+
+
+
+
```
Comment (EmOlsz) [inline code/style.css]: unnecessary blank lines

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Good job Nathalie, congratulations on well done project :)
I would like you to check out both Claudia's and my comment and fix your code accordingly to them. After that, project will be approved!
Moreover, I would like you to fix logic of your application - right now it's possible to change suspect, weapon and room card value even after we reveal crime - it should be disabled

**Comment by nathalienordstrom** [comment]
a

**Comment by nathalienordstrom** [comment]
Have I not fixed all that you wanted or what's left?

**Comment by EmOlsz** [comment]
I might have missed some notifications about your changes, sorry.
Project is approved :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**